qiskit/aqua/utils/run_circuits.py

```diff
@@ -342,16 +342,7 @@ def run_on_backend(backend, qobj, backend_options=None,
                    noise_config=None, skip_qobj_validation=False):
     """ run on backend """
     if skip_qobj_validation:
-        job_id = str(uuid.uuid4())
         if is_aer_provider(backend):
-            # pylint: disable=import-outside-toplevel
-            try:
-                from qiskit.providers.aer.aerjob import AerJob
-            except ImportError as ex:
-                raise MissingOptionalLibraryError(
-                    libname='qiskit-aer',
-                    name='run_on_backend',
-                    pip_install='pip install qiskit-aer') from ex
             temp_backend_options = \
```

Comment:
I think this can be simplified in a way that will work for Aer 0.7 and earlier verisons. You can try:
```suggestion
           if backend_options is not None:
                for option, value in backend_options.items():
                    if option == 'backend_options':
                        for key, val in value.items():
                            setattr(qobj.config, key, val)
                    else:
                        setattr(qobj.config, option, value)
            if noise_config is not None and 'noise_model' in noise_config:
                qobj.config.noise_model = noise_config['noise_model']
            job = backend.run(qobj, validate=False)
```